src/cnxml/glossary.js

```diff
@@ -36,9 +36,13 @@ function se_meaning(obj, children) {
         <meaning>
             {meaning_children}
         </meaning>
-        <example>
-            {example.props.children}
-        </example>
+        {
+            examples.map(e => {
+                return <example>
+                    {e.props.children}
+                </example>
+            })
```

Comment:
What is this map for? Shouldn't `examples` already be wrapped in `<example>`?

Reply:
:man_facepalming: True